playground/src/email-loader/email-loader.js

```diff
@@ -12,7 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import events from '../events/events.js';
 import * as quotedPrintable from 'quoted-printable';
+import * as FileUpload from '../file-upload/file-upload.js';
```

Comment:
The email loader should not have any dependencies to the file upload or the other way round. They should be coupled in `Importer`.